src/main/java/sirius/db/jdbc/Capability.java

```diff
@@ -58,7 +59,15 @@ public enum Capability {
     /**
      * Signales that the database supports DECIMAL fields.
      */
-    DECIMAL_TYPE;
+    DECIMAL_TYPE,
+
+    /**
+     * Signals that the database performs auto-commits, therefore no explicit calls to {@link Connection#commit()}
+     * should be performed.
+     * <p>
+     * This is especially needed for Clickhouse, as newer drivers uses the commit method to control transactions.
+     */
+    AUTO_COMMIT;
```

Comment:
shouldn't we invert this to TRANSCATION since actually clickhouse does not support them? (s.b.)